### DISCUSSION TITLE
Switching over to x.com seems to have broken the control panel

### DISCUSSION THREAD

**Discussion by VexWerewolf** [insin/control-panel-for-twitter#570] [General]
Elon's petulant insistence on renaming the site seems to have broken the extensions that make it useable.

**Comment by Sigge310** [comment]
Can confirm. Doesn't work on firefox anymore

**Comment by tipsheda** [comment]
Yep, also on Firefox and it's not working anymore with the switch to x.com.

Edit: saw somewhere that version 4.0 should have x.com support and after removing and reinstalling the extension, it works again for me with the new domain switch.
